Traceback (most recent call last):
  File "pgNew.py", line 174, in <module>
    if __name__=="__main__": startTime = time.time(); exp()
  File "pgNew.py", line 166, in exp
    createDCTableSetup(sys.argv[2], int(sys.argv[3]),int( sys.argv[4]), int(sys.argv[5]), int(sys.argv[6]))
  File "pgNew.py", line 43, in createDCTableSetup
    createTable(cur, conn, 'dc_' + table, 6, 1)
  File "pgNew.py", line 33, in createTable
    cur.execute("CREATE TABLE " + name + " " + cols)
psycopg2.ProgrammingError: relation "dc_exp" already exists

Traceback (most recent call last):
  File "pgNew.py", line 174, in <module>
    if __name__=="__main__": startTime = time.time(); exp()
  File "pgNew.py", line 168, in exp
    createDCTableLevel1(sys.argv[2], int(sys.argv[3]),int( sys.argv[4]), int(sys.argv[5]), int(sys.argv[6]))
  File "pgNew.py", line 67, in createDCTableLevel1
    + " OFFSET " + str(x*sizeChunk) + ") as foo")
NameError: name 'j' is not defined
Traceback (most recent call last):
  File "pgNew.py", line 174, in <module>
    if __name__=="__main__": startTime = time.time(); exp()
  File "pgNew.py", line 170, in exp
    createDCTableLevel2(sys.argv[2], int(sys.argv[3]),int( sys.argv[4]), int(sys.argv[5]), int(sys.argv[6]))
  File "pgNew.py", line 106, in createDCTableLevel2
    + " OFFSET " + str(c*sizeChunk) + ") as foo")
psycopg2.ProgrammingError: column "col" does not exist
LINE 1: SELECT CORR(x, y) FROM (SELECT cast(col as double precision)...
                                            ^

Traceback (most recent call last):
  File "pgNew.py", line 174, in <module>
    if __name__=="__main__": startTime = time.time(); exp()
  File "pgNew.py", line 172, in exp
    createDCTableLeveln(sys.argv[2], int(sys.argv[3]),int( sys.argv[4]), int(sys.argv[5]), int(sys.argv[6]))
  File "pgNew.py", line 125, in createDCTableLeveln
    for x in (numCols):
TypeError: 'int' object is not iterable
